cinch/playbooks/jenkins_restart_utility.yml

```diff
@@ -0,0 +1,148 @@
+# This playbook aims to gather diagnostics data in the event of
+# trouble with Jenkins, and then restarts Jenkins to keep operations running.
+# By default, only the diagnostics data is collected, but the Jenkins service
+# restart can be enabled by setting the 'restart_jenkins' variable to true:
+
+# Example:
+# ansible-playbook -u root --private-key /path/to/key
+# -i '10.0.183.169', jenkins_restart_utility.yml -e "restart_jenkins=true"
+
+- hosts: all
+  vars:
+    jenkins_pid_path: '/run/jenkins.pid'
+    diag_path_prefix: '/var/lib/jenkins/cinch_diagnostics'
+    jenkins_log_dir: '/var/log/jenkins'
+    log_history_days: '-7d'
+    jstack_path: '/usr/bin/jstack'
+    jmap_path: '/usr/bin/jmap'
+    jstat_path: '/usr/bin/jstat'
+    java_path: '/usr/bin/java'
```

Comment:
I don't disagree, but I did find that some other parts of cinch do use vars for the full path to some java programs, so at least we're consistent.